oop/music/beats.py

```diff
@@ -0,0 +1,11 @@
+from oop.music.music import Music
+
+class Beats(Music):
+    """ a class that identifies Beats in Music """
+    def __init__(self, title: str, bpm: int, instruments: [], no_lyrics: bool):
+        super().__init__(title, bpm, instruments, False)
+        self.no_lyrics = no_lyrics 
```

Comment:
if the no_lyrics member is always false, there's no need to include it in the __init__() constructor. no point deductions; we can discuss this in class if you'd like.